bindata/network/openshift-sdn-azure/azure-drop-icmp-daemonset-sdn.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: add-iptables
```

Comment:
you need to remove the `azure-drop-icmp-daemonset-sdn.yaml` file from the PR since it's no longer used (right?)